sonic_service_client/dbus_client.go

```diff
@@ -114,7 +114,7 @@ func (c *DbusClient) ApplyPatchYang(patch string) error {
 	busName := c.busNamePrefix + modName
 	busPath := c.busPathPrefix + modName
 	intName := c.intNamePrefix + modName + ".apply_patch_yang"
-	err := DbusApi(busName, busPath, intName, 60, patch)
```

Comment:
let's set to 240 to be same with internal limit

Reply:
Fixed